ui/src/components/NavigationBar/MainMenu.js

```diff
@@ -137,17 +137,16 @@ const MainMenu = ({ login, onClose }) => {
                 >
```

Comment:
<img width="681" alt="image" src="https://user-images.githubusercontent.com/24607458/183419863-394f63ae-ea93-44cb-a42e-a2f5b46cc0b3.png">
<img width="475" alt="image" src="https://user-images.githubusercontent.com/24607458/183419893-7b6d3da4-bcff-4efe-a6ba-a5fbdade1774.png">

color of the bg looks different, may need to check with cxo if they want to change the bg image color